atc/creds/keyvault/manager_test.go

```diff
@@ -0,0 +1,53 @@
+package keyvault
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+func TestHealth(t *testing.T) {
+	t.Run("a working client should return good health", func(t *testing.T) {
```

Comment:
We typically use [`ginkgo`](https://github.com/onsi/ginkgo) for tests but I won't split hairs. (More info: https://github.com/concourse/concourse/blob/master/CONTRIBUTING.md#testing-your-changes)

Reply:
Yeah, I wasn't sure on that because some of them (I think the vault one) didn't use ginkgo. I'll change it over so it doesn't diverge from the core project